world/dominion/general_dominion_commands.py

```diff
@@ -2514,8 +2513,12 @@ def func(self):
                 if num >= max_p:
                     caller.msg("You already have the maximum number of proteges for your social rank.")
                     return
-                if not self.check_social_rank_difference(char):
+                rankcheck, diff = self.check_social_rank_difference(caller.db.char_ob,char)
```

Comment:
```suggestion
                rankcheck, diff = self.check_social_rank_difference(caller.char_ob,char)
```
Getting rid of the `.db.char_ob` references - characters made after this point won't even have the Attribute set, so have to use the property.